index.html

```diff
@@ -1885,6 +1887,40 @@ <h2>
           shipping option. <a>User agents</a> MAY indicate pending fields in
           the user interface for the payment request.
         </dd>
+        <dt>
+          <dfn>type</dfn> member
+        </dt>
+        <dd>
+          A <a>PaymentItemType</a> enum value. A user agent MAY use the value
+          of <a>type</a> to assist in the presentation of <a>PaymentItem</a>s
+          by, for example, visually grouping types together or otherwise
+          distinguishing them from other types (or from items that have no
+          associated type).
+        </dd>
+      </dl>
+    </section>
+    <section data-dfn-for="PaymentItemType">
+      <h2>
+        <dfn>PaymentItemType</dfn> enum
+      </h2>
+      <pre class="idl">
+        enum PaymentItemType {
+          "tax"
```

Comment:
Instead of `other`, could we have `lineItem` as the default and include in the enum?

Reply:
I'm not sure we need that default at all.